### DISCUSSION TITLE
Is there a way to insert emoji in a text generated by ImageMagick?

### DISCUSSION THREAD

**Discussion by ProgrammingLife** [ImageMagick/ImageMagick#5268] [General]
I mean something like that:
```
magick -background navy -fill yellow -font Ubuntu-bold -pointsize 100 -strokewidth 5 -stroke black -size 1000x500 -kerning -5 -interline-spacing -40 -gravity west label:"text |😀|  text" output.png
```

Is there a chance to make it work?

**Comment by fmw42** [comment]
None that I know about.

**Comment by urban-warrior** [comment]
We leverage the Freetype delegate library to render text.  Freetype can support emoji's but it requires a challenging patch to ImageMagick which we have investigated, but not deployed.  We do not have an ETA on when we will complete emoji support.
